src/content/app/genome-browser/components/track-settings-panel/TrackSettingsPanel.scss

```diff
@@ -52,3 +57,11 @@
     vertical-align: middle;
   }
 }
+
+.moveTracksIcon {
```

Comment:
`.moveTracksIcon` -> `reorderTrack`?

- we are dealing here with a single track, not multiple tracks
- the purpose of this element will be to change track order